include/userobjects/PICStudyBase.h

```diff
@@ -0,0 +1,52 @@
+//* This file is part of FENIX: Fusion Energy Integrated Multiphys-X,
+//* A multiphysics application for modeling plasma facing components
+//* https://github.com/idaholab/fenix
+//*
+//* FENIX is powered by the MOOSE Framework
+//* https://www.mooseframework.inl.gov
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Function.h"
+#include "ClaimRays.h"
+#include "RayTracingStudy.h"
+#include "VelocityUpdaterBase.h"
+
+class PICStudyBase : public RayTracingStudy
+{
+public:
+  PICStudyBase(const InputParameters & parameters);
+
+  static InputParameters validParams();
+
+  virtual void generateRays() override;
+  // The banked rays to be used on the next timestep (restartable)
+  std::vector<std::shared_ptr<Ray>> & _banked_rays;
+
+protected:
+  virtual void postExecuteStudy() override;
+  /** Ray Data for storing all components of velocity */
+  const RayDataIndex _v_x_index;
+  const RayDataIndex _v_y_index;
+  const RayDataIndex _v_z_index;
```

Comment:
```suggestion
  /// Ray data for storing velocity components
  ///@{
  const RayDataIndex _v_x_index;
  const RayDataIndex _v_y_index;
  const RayDataIndex _v_z_index;
  ///@}
```

When you wrap in `///@{  ...  ///@}`, they show up grouped in doxygen